server/src/weaverbird/backends/mongo_translator/mongo_pipeline_translator.py

```diff
@@ -0,0 +1,25 @@
+from weaverbird.backends.mongo_translator.steps import mongo_step_translator
+from weaverbird.pipeline import Pipeline, PipelineStep
+
+
+def translate_pipeline(
+    pipeline_to_translate: Pipeline,
+) -> list:
```

Comment:
`list[dict[str, Any]]` to be more precise?

Reply:
This will make mypy check that each step function return a compatible type too :)